.github/workflows/maven.yml

```
@@ -103,6 +103,7 @@ jobs:
           semantic-release --prepare @conveyal/maven-semantic-release --publish @semantic-release/github,@conveyal/maven-semantic-release --verify-conditions @semantic-release/github,@conveyal/maven-semantic-release --verify-release @conveyal/maven-semantic-release --use-conveyal-workflow --dev-branch=dev --skip-maven-deploy
       # The git commands get the commit hash of the HEAD commit and the commit just before HEAD.
       - name: Prepare deploy artifacts
+        if: github.event_name != 'pull_request'
```

My only comment would that this can sometimes be the kind of random CI tweak that gets lost to the sands of time. So I might inject a comment here explaining why we're skipping. Also, I don't really expect us to have this workflow run on other events besides `push` and `pull_request`, but a more precise check might actually be `github.event_name == 'push'` (in case other events get added here and we don't want deploy to run).

```suggestion
        # Only deploy on push (pull_request will deploy a temp. merge commit. See #400.)
        if: github.event_name == 'push'